### PR TITLE
feat: add LVLHGD and LVLHGDGT local orbital frames

### PR DESCRIPTION
**Pull request by vishwa2710**
- LVLHGD (Geodetic)
- LVLHGDGT (Geodetic + aligned with ground track)
- Tests to be added.

### CODE REVIEW

File: src/OpenSpaceToolkit/Astrodynamics/Trajectory/Orbit.cpp [hunk 1]
```diff
@@ -402,32 +409,28 @@ Shared<const Frame> Orbit::getOrbitalFrame(const Orbit::FrameType& aFrameType) c
...
-                const Duration delta = Duration::Seconds(1.0);  // TBM This should be a parameter
+                const Duration delta = Duration::Seconds(0.1);  // TBM This should be a parameter
 
-                const Quaternion q_VVLH_next_GCRF =
+                const Quaternion q_LOF_GCRF_next =
                     anAttitudeGenerator(this->getStateAt(anInstant + delta).inFrame(Frame::GCRF()));
 
-                const Quaternion q_VVLH_next_VVLH = (q_VVLH_next_GCRF * q_VVLH_GCRF.toConjugate()).toNormalized();
-                const RotationVector rv_VVLH_next_VVLH = RotationVector::Quaternion(q_VVLH_next_VVLH);
+                const Quaternion q_LOF_next_LOFT = (q_LOF_GCRF_next * q_LOF_GCRF.toConjugate()).toNormalized();
```
Comment (lucas-bremond) [inline src/OpenSpaceToolkit/Astrodynamics/Trajectory/Orbit.cpp:421]:
```suggestion
                const Quaternion q_LOF_next_LOF = (q_LOF_GCRF_next * q_LOF_GCRF.toConjugate()).toNormalized();
```

?

**Review by Derollez — APPROVED** [review]
Super clean! LGTM

File: src/OpenSpaceToolkit/Astrodynamics/Trajectory/Orbit.cpp (hunk 1 above)
Comment (apaletta3) [reply src/OpenSpaceToolkit/Astrodynamics/Trajectory/Orbit.cpp:421]: Its an easter egg, lucas ;)

**Review by apaletta3 — APPROVED** [review]
Nice!

File: test/OpenSpaceToolkit/Astrodynamics/Trajectory/Orbit.test.cpp [hunk 2]
```diff
@@ -764,21 +767,14 @@ TEST(OpenSpaceToolkit_Astrodynamics_Trajectory_Orbit, GetPassWithRevolutionNumbe
 
 TEST(OpenSpaceToolkit_Astrodynamics_Trajectory_Orbit, GetOrbitalFrame)
```
[the remark this answers is not in the document]
Comment (vishwa2710) [reply test/OpenSpaceToolkit/Astrodynamics/Trajectory/Orbit.test.cpp:768]: So if we split this into smaller tests, then we would need to duplicate all the test code... which is fairly boilerplate. Currently the only difference between the tests is the file it's reading, and the tolerances.

[the remark this answers is not in the document]
Comment (lucas-bremond) [reply test/OpenSpaceToolkit/Astrodynamics/Trajectory/Orbit.test.cpp:768]: ok!

**Pull request merged**